Makefile

```diff
@@ -73,13 +89,14 @@ else
 endif
 
 install_protoc:
-ifdef PROTOC_VERSION
 ifeq (,$(findstring $(PROTOC_VERSION_NEEDED), $(PROTOC_VERSION)))
 	@echo "⚠️  protoc version needed: $(PROTOC_VERSION_NEEDED) vs $(PROTOC_VERSION) installed"
-	@echo "ℹ️  Please download the correct protobuf binary for your OS from https://github.com/protocolbuffers/protobuf/releases/tag/v${PROTOC_VERSION_NEEDED}"
-	@echo "ℹ️  Note the download's name will look like this: protoc-${PROTOC_VERSION_NEEDED}-osx-x86_64.zip"
-	@echo "ℹ️  Copy the unzipped folder into this project, and rename it to \"protoc\""
+	@echo "ℹ️  Vendoring protoc $(PROTOC_VERSION_NEEDED)"
+  
+	@mkdir -p tmp
+	curl --location --output tmp/$(PROTOC_FILENAME) $(PROTOC_URL)
+	unzip -d protoc tmp/$(PROTOC_FILENAME)
+	rm tmp/$(PROTOC_FILENAME)
+  
 	@echo "ℹ️  If this is macOS, you will need to try running the binary yourself, then go to Security & Privacy to explicitly allow it."
```

Comment:
Definitely not in the scope of this PR but I realized that we have this line, but we kinda addressed this in https://github.com/pganalyze/collector/pull/667 so maybe we can remove this line.

Reply:
This is talking about the `protoc` binary, though, right? Wasn't #667 was about the generated collector binary?

Reply:
ahhh good call! well ok let's leave it as is